firestore-perspective-toxicity/README.md

```diff
@@ -0,0 +1,13 @@
+# Analyze Toxicity with Perspective API
+
+**Author**: Perspective API (**[https://perspectiveapi.com](https://perspectiveapi.com/)**)
+
+**Description**: We’ve partnered with the Perspective team to build the Analyze Toxicity extension, which leverages Google’s ML to classify the level of toxicity, threat and profanity of your user comments. The Perspective team uses the power of Google’s machine learning to help mitigate toxicity and ensure healthy dialogue online. It is trusted by sites like the New York Times or the Wall Street Journal. Please check it out [here](https://github.com/conversationai/firestore-perspective-toxicity).
```

Comment:
Let's remove the phrase "Google's ML" and "Google's machine learning", as Google has a ton of different machine learning work, and this could be misleading. We can replace with just "machine learning"?

Reply:
SGTM